### DISCUSSION TITLE
Email notifications disabe ssl

### DISCUSSION THREAD

**Discussion by andrusstrockiy** [argoproj-labs/argocd-notifications#314] [Q&A]
Followed https://blog.argoproj.io/notifications-for-argo-bb7338231604 unfortunately i was unable to send through our local smtp server due to ssl error

```bash
time="2021-07-21T13:13:31Z" level=info msg="Sending notification about condition 'on-sync-succeeded.[0].zxM90Et6k4Elb1-fHdjtDJq0xR0' to '{gmail myemail@mail.com}'" app=argocd/eq-edo-srv
time="2021-07-21T13:13:32Z" level=error msg="Failed to notify recipient {gmail myemail@mail.com} defined in app argocd/asts-eq-edo-service: x509: certificate signed by unknown authority" app=argocd/eq-edo-srv
```
which is expected cause our smtp server configured on port 25 so far i couldn't find any information on how to disable ssl check 
https://argocd-notifications.readthedocs.io/en/stable/services/email/#email 
So my question is there is an option on disabling such check ?

**Comment by ryota-sakamoto** [comment]
Hi @andrusstrockiy 
You can use `insecure_skip_verify` field to disable ssl check.

**Reply by andrusstrockiy** [reply]
@ryota-sakamoto 
my guess that configmap should look as following 
` service.email.gmail: >-
    { username: $email-username, host: smtpauth.mail.com, port: 25, from:
    osa@mail.com, password: $email-password, insecure_skip_verify: true }`
   cause with 
   `    { username: $email-username, host: smtpauth.mail.com, port: 25, from:
    osa@mail.com, password: $email-password, insecure_skip_verify }`
    I got the same error as in previous post
